ontology/yaml/resources/units/units.yaml

```diff
@@ -12,58 +12,315 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+acceleration:
+- meters_per_second_per_second: STANDARD
+
+#adimensional:
+#- no_units: STANDARD
+
+angle:
+- degrees_angular: STANDARD
+- radians
+
+angularmomentum:
+- joule_seconds: STANDARD
+
+# apparentpower:
+# - volt_amperes: STANDARD
+# - kilovolt_amperes
+# - mega_volt_amperes
+
+area:
+- square_meters: STANDARD
+- square_centimeters
+- square_feet
+- square_inches
+
 concentration:
 - parts_per_million: STANDARD
+- parts_per_billion
+
 current:
 - amperes: STANDARD
 - milliamperes
+
+density:
+- kilogram_per_cubic_meter: STANDARD
+
+electriccapacitance:
+- farads: STANDARD
+
+electricconductance:
+- siemens: STANDARD
+
+electricconductivity:
+- siemens_per_meter: STANDARD
+
+electriccurrentdensity:
+- amperes_per_square_meter: STANDARD
+
+electricfield:
+- volts_per_meter: STANDARD
+
+electricinductance:
+- henrys: STANDARD
+
+electricphasedifference:
+- degrees_phase: STANDARD
+
+electromagneticfieldstrength:
+- amperes_per_meter: STANDARD
+
+electromagneticmoment:
+- ampere_square_meters: STANDARD
+
+electricresistivity:
+- ohm_meters: STANDARD
+
 energy:
 - joules: STANDARD
 - kilowatt_hours
+- btus
+- kilobtus
```

Comment:
Should the kilo* and mega* units be consistent with underscores?  e.g. this has kilobtus and mega_btus.
Should it be one of:
- kilo_btus and mega_btus or
- kilobtus and megabtus

Reply:
thanks for spotting this, for consistency all metric prefixes have been replaced without the underscore